config/ampache.cfg.php.dist

```diff
@@ -756,12 +756,18 @@ site_charset = UTF-8
 ;#########################################################
 
 ; This setting will silently create an ampache account
-; for anyone who can login using ldap (or any other login
+; for anyone who can login using LDAP (or any other login
 ; extension). The default is to create new users as guests
 ; see auto_user config option if you would like to change this
 ; DEFAULT: false
 ;auto_create = "false"
 
+; This setting will silently update an ampache account's
+; informations for anyone who can login using LDAP
+; (or any other login extension).
+; DEFAULT: false
+;auto_update = "false"
```

Comment:
I totally agree. I kept that name for coherence with the `auto_create` setting and because I didn't have any other idea.

Thing is, this setting would affect any external auth and not only LDAP.

So maybe one of these:

- `ldap_auto_update`
- `auto_update_external`
- `external_auto_update`
- `auto_update`
- ...

What do you think?
-- 
Niols

Le 1 mai 2016 13:38:55 GMT+02:00, Afterster <notifications@github.com> a écrit :
>>  ; extension). The default is to create new users as guests
>>  ; see auto_user config option if you would like to change this
>>  ; DEFAULT: false
>>  ;auto_create = "false"
>>  
>> +; This setting will silently update an ampache account's
>> +; informations for anyone who can login using LDAP
>> +; (or any other login extension).
>> +; DEFAULT: false
>> +;auto_update = "false"
>
>Good to have an option for this but it can mislead administrators
>because of the other `autoupdate` setting. Would be better to rename it
>or prefix it with `ldap_`
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/ampache/ampache/pull/1208/files/07dd89e12f957634c66b24597e89d535be6f833c#r61685904